### ISSUE TITLE
Frontend reports services as OK but it's failed

### ISSUE THREAD

**Issue by GitHK** [ITISFoundation/osparc-simcore#3885]
Unfortunately I have no more useful logs for the frontend. But an error occurreed in the backend and the frontend reported it as blue OK, once clicked the services is unavailable. Of course reloading it still does not work.

I would expect the following to happen:
1. The frontend should not report this as OK
2. if the iframe says 500 or some other error I would just mark it as failed. The user has a possibility to restart it cleanly

![Screenshot 2023-02-16 at 07 50 05](https://user-images.githubusercontent.com/5694077/219289774-dfd50c51-5633-473a-8f3b-eea9063ceaf6.png)

**Comment by odeimaiz** [comment]
@GitHK If the frontend says that the service is ready is because the backend said it is all good and you can connect. From that moment on the frontend stops pulling the state....

I would wait for the backend-pushes-services-status feature to "fix" this.

**Comment by sanderegg** [comment]
@GitHK I agree with @odeimaiz . We need to push the state of the service into a rabbitMQ channel and push it to the frontend. this way there is no more polling mismatch.

**Comment by GitHK** [comment]
I actually know why this is happening like this. It has something to do with the project networks failing #3866. They get attached just afte the fronted mark container as ready.
And @sanderegg will solve the issue, I would also go for that.
